src/main/java/mate/academy/boot/bootdemo/service/FileParser.java

```diff
@@ -0,0 +1,8 @@
+package mate.academy.boot.bootdemo.service;
+
+import java.util.List;
+import mate.academy.boot.bootdemo.model.dto.UserDto;
+
+public interface FileParser {
```

Comment:
let's make this interface more abstract by adding parameter
```suggestion
public interface FileParser<T> {
```